File: AndroidClient/LdpAndroidClient/app/src/main/java/com/ldp/androidclient/network/ILdpPacketHandler.java
package com.ldp.androidclient.network;

import com.ldp.androidclient.disposable.ILdpDisposable;
import com.ldp.androidclient.protocol.LdpProtocol.*;

public interface ILdpPacketHandler extends ILdpDisposable {
    void handle(LdpPacket packet);
}

File: Server/Server/Network/Handlers/PacketHandlerBase/ILdpPacketHandler.cs
using Server.Protocol;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Server.Network.Handlers.PacketHandlerBase
{
    interface ILdpPacketHandler : IDisposable
    {
        void Handle(LdpPacket packet);
    }
}
